test/auth_cluster/suite.yaml

```diff
@@ -0,0 +1,8 @@
+type: Topology
+pool_size: 2
+cluster:
+  initial_size: 0
+extra_scylla_config_options:
+  authenticator: PasswordAuthenticator
+  authorizer: CassandraAuthorizer
+  permissions_validity_in_ms: 3600s
```

Comment:
We override these settings in the test body - https://github.com/scylladb/scylladb/pull/15922/files#diff-d1c566a12ba904e3cf20de29fdcf08d52be64b3e8b6b1bf6daff8b3a38b04d04R28-R31 - so maybe they could be just remove from here?